axelor-human-resource/src/main/java/com/axelor/apps/hr/service/expense/ExpenseLineCreateServiceImpl.java

```diff
@@ -230,4 +241,15 @@ protected void setActiveCompanyCurrency(ExpenseLine expenseLine) {
       }
     }
   }
+
+  @Override
+  public List<Employee> getEmployeeList(
+      List<Long> employeeIdList, ExpenseLine expenseLine, LocalDate expenseDate) {
+    if (employeeIdList == null) {
+      return null;
+    }
+    return expenseLineService.getEmployeeDomain(expenseDate).stream()
+        .filter(el -> employeeIdList.contains(el.getId()))
+        .collect(Collectors.toList());
+  }
```

Comment:
The name of this method should be more specific like filterInvitedCollaborators. Also it could be placed in ExpenseLineToolService since it is not only used in creation but also in update. It would also be better to take in parameter a List of employee instead of a List of Long.